style.css

```diff
@@ -1,14 +1,24 @@
 h1, h2 {
+
   font-family: sans-serif;
   border-style: dotted;
   border-width: thin;
   position: relative;
-  color:yellow;
-  background-color: green;
+  color:blue;
+
```

Comment:
again on the whitespace (not something that you'll get points taken away for on this assignment, but we'll dock points in future projects because we want you to stick to best-practices)